larpandoracontent/LArObjects/LArMCParticle.h

```diff
@@ -103,6 +121,15 @@ inline LArMCParticle::LArMCParticle(const LArMCParticleParameters &parameters) :
     object_creation::MCParticle::Object(parameters),
     m_nuanceCode(parameters.m_nuanceCode.Get())
 {
+    for (auto const &mcStepPosition : parameters.m_mcStepPositions)
```

Comment:
Could these two vectors be initialised in the initialiser list? e.g.
`m_mcStepPositions(parameters.m_mcStepPositions)`